awesome_owl/static/src/card/card.js

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@odoo/owl'
+
+export class Card extends Component{
+    static template = "awesome_owl.Card";
+    static props = {
+        title:String,
+        slots:{
+            type:Object,
+            shape:{
+                default:true
+            }
+        }
+    }
```

Comment:
```suggestion
    };
```
Use `;` for variable not a function